### PR TITLE
Bug/cidrnotvalid

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg @jframos

### CODE REVIEW

**Comment by flopezag** [comment]
Some previous comments from the closed PR should be taken into account...

**Comment by hmunfru** [comment]
comments were already taken into account https://github.com/telefonicaid/fiware-paas/pull/155

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java [hunk 1]
```diff
@@ -75,18 +78,18 @@ public boolean exists (ClaudiaData claudiaData, NetworkInstance networkInstance,
...
-                     networkInstance.getIdNetwork() + " does no exists in Openstack") ;
+            	log.warn ("The network "+  networkInstance.getNetworkName()+ " with id "
+                    + networkInstance.getIdNetwork() + " does no exists in Openstack") ;
             	this.deleteInDb(networkInstance);
             	return false;
-            	
+
             } else {
             	log.info("The network "+  networkInstance.getNetworkName()+ " with id " +
                     networkInstance.getIdNetwork() + " already exists");
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java]: + should be in the next line.

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java [hunk 2]
```diff
@@ -224,18 +229,21 @@ public void delete(ClaudiaData claudiaData, NetworkInstance networkInstance, Str
...
         }
         try {
-            networkClient.deleteNetworkToPublicRouter(claudiaData, networkInstance, region);
+            networkClient.deleteNetworkToPublicRouter(claudiaData,
+                networkInstance, region);
         } catch (Exception e) {
-            log.warn ("It is not possible to delete the public interface. It is not exist: "+
-                e.getMessage());
+            log.warn ("It is not possible to delete the public interface. " +
+                "It is not exist: " + e.getMessage());
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java]: + should be in the next line

File: core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java [hunk 3]
```diff
@@ -297,22 +309,34 @@ public boolean canBeDeleted(ClaudiaData claudiaData, NetworkInstance networkInst
             for (Port port : ports) {
                 strPorts = strPorts + " " + port.getNetworkId();
             }
-            log.info("It is not possible to undeply the network since there are VMs associated to it " + strPorts);
+            log.info("It is not possible to undeply the network since there " +
```
Comment (flopezag) [inline core/src/main/java/com/telefonica/euro_iaas/paasmanager/manager/impl/NetworkInstanceManagerImpl.java]: + should be in the next line

**Comment by jesuspg** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**